server/routes/creatingLicences/handlers/checkAnswers.ts

```diff
@@ -27,11 +28,22 @@ export default class CheckAnswersRoutes {
       )
     }
 
-    const { conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
+    let { conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
       licence.additionalLicenceConditions
     )
 
-    res.render('pages/create/checkAnswers', { additionalConditions, conditionsWithUploads, backLink })
+    if (isInPssPeriod) {
+      ;({ conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
```

Comment:
you can avoid the mutable reference (`let`) by restructuring? 
```ts
const conditionsToDisplay  = !isInPssPeriod 
  ?  licence.additionalLicenceConditions
  :  (await this.licenceService.getParentLicenceOrSelf(licence.id.toString(), user))?.additionalLicenceConditions
```

Reply:
You're tests aren't exercising this conditional though

Reply:
added tests to check this condition